layout/src/misc.css

```diff
@@ -20,6 +20,11 @@ dockit-layout :where(h1, h2, h3, h4, h5, h6) {
   scroll-margin-top: calc(var(--private--dockit-layout-header-height) + 1rem);
 }
 
+dockit-layout :where(figcaption) {
```

Comment:
you need `.prose figcaption`, because you are providing styles for content which is inside `.prose`
any specific reason you want `:where`? I think just tag name will do

Reply:
btw, existing selector `dockit-layout :where(h1, h2, h3, h4, h5, h6)` is wrong, should be `.prose :where(h1, h2, h3, h4, h5, h6)` too :) please do not copy my mistake

Reply:
fixed all